l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -642,7 +673,7 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
         uint16 _l2TxNumberInBatch,
         bytes32[] calldata _merkleProof
     ) external override {
-        bytes32 assetInfo = bytes32(uint256(uint160(_l1Asset)));
+        bytes32 assetInfo = IL1NativeTokenVault(nativeTokenVault).getAssetInfo(_l1Asset);
```

Comment:
NTV address is already IL1NativeTokenVault, no need to change types

Reply:
done